scripts/travis/build-all-in-one-image.sh

```diff
@@ -40,7 +40,23 @@ upload_to_docker() {
 
 make build-all-in-one GOOS=linux GOARCH=$GOARCH
 repo=jaegertracing/all-in-one
-docker build -f cmd/all-in-one/Dockerfile -t $repo:latest cmd/all-in-one --build-arg TARGETARCH=$GOARCH
+docker build -f cmd/all-in-one/Dockerfile \
+        --target release \
+        --tag $repo:latest cmd/all-in-one \
+        --build-arg base_image=localhost/baseimg:1.0.0-alpine-3.12 \
```

Comment:
Not sure we have everything tied correctly: this image is created via `create-baseimg`, which isn't being called when building the all-in-one:

https://github.com/jaegertracing/jaeger/blob/3ae29da424085fda3dcafbd42f0f0fc5f107e945/.travis.yml#L70

Perhaps we need a Makefile target to build the all-in-one, which builds the base/debug images and calls this script?

Reply:
Thank you for catching it. I've updated the PR with a fix. Hopefully it will solve the issue.